cantusdb/csv/fetch.py

```diff
@@ -0,0 +1,32 @@
+"""
+fetch the CSV exports
+"""
+
+import json
+import os
+import time
+import requests
+
+# URL of the file to download
+SOURCE_PATH = os.path.join(
+    os.path.dirname(__file__), "../data/mappings/sources_short.json"
```

Comment:
you don't explain anywhere what "sources_short.json" is....

Reply:
It's in README.